acis_thermal_check/main.py

```diff
@@ -1084,7 +1110,7 @@ def make_validation_plots(self, tlm, model_spec, outdir):
         for plot in plots.values():
             for key in plot:
                 if key in ['lines', 'hist']:
-                    outfile = outdir / plot[key]['filename']
+                    outfile = outdir / plot[key]["filename"]
```

Comment:
Elsewhere you are using the attribute .filename instead.

Reply:
We can't use the attribute in the validation plots yet--I deemed it too much for this PR